pkg/resolve/resolver.go

```diff
@@ -7,6 +7,8 @@ import (
 	"path/filepath"
 	"reflect"
 	"strings"
+
+	"github.com/jkcfg/jk/pkg/record"
```

Comment:
To avoid this import, you could define the required interface here

Reply:
Not sure it brings us much here as we not only use a function that could be indeed abstracted away with an interface but also types and constants defined in that package. I guess it means those two packages are coupled. I'm sure we can revisit that later™ if needed :)